api.go

```diff
@@ -0,0 +1,29 @@
+package messagebird
+
+import (
+	"net/url"
+	"strconv"
+)
+
+// PaginationRequest can be used to set pagination options in List().
+type PaginationRequest struct {
+	Limit, Offset int
+}
+
+func (cpr *PaginationRequest) QueryParams() string {
+	if cpr == nil {
```

Comment:
It won't panic because this check, we have one test which approve it.
It is [TestListWebhooks.all](https://github.com/vallerion/go-rest-api/blob/sync-sdk/conversation/webhook_test.go#L63) call ListWebhooks with nil on PaginationRequest param.